docs(release): update 1.7.2 release note (#9680)
longhorn/longhorn: CHANGELOG/CHANGELOG-1.7.2.md

## Longhorn v1.7.2 Release Notes

> Longhorn 1.7.2 introduces several improvements and bug fixes that are intended to improve system quality, resilience, and stability.

Reviewer: ```suggestion
Longhorn 1.7.2 introduces several improvements and bug fixes that are intended to improve system quality, resilience, stability, and security. 
```

## Installation

>  [!IMPORTANT]
**Ensure that your cluster is running Kubernetes v1.21 or later before installing Longhorn v1.7.2.**

You can install Longhorn using a variety of tools, including Rancher, Kubectl, and Helm. For more information about installation methods and requirements, see [Quick Installation](https://longhorn.io/docs/1.7.2/deploy/install/) in the Longhorn documentation.

## Upgrade

>  [!IMPORTANT]
**Ensure that your cluster is running Kubernetes v1.21 or later before upgrading from Longhorn v1.6.x or v1.7.x (< v1.7.0) to v1.7.2.**

Longhorn only allows upgrades from supported versions. For more information about upgrade paths and procedures, see [Upgrade](https://longhorn.io/docs/1.7.2/deploy/upgrade/) in the Longhorn documentation.

## Deprecation & Incompatibilities

The functionality of the [environment check script](https://github.com/longhorn/longhorn/blob/v1.7.x/scripts/environment_check.sh) overlaps with that of the Longhorn CLI, which is available starting with v1.7.0. Because of this, the script is deprecated in v1.7.0 and is scheduled for removal in v1.8.0.

> For information about important changes, including feature incompatibility, deprecation, and removal, see [Important Notes](https://longhorn.io/docs/1.7.0/deploy/important-notes/) in the Longhorn documentation.

Reviewer: ```suggestion
For information about important changes, including feature incompatibility, deprecation, and removal, see [Important Notes](https://longhorn.io/docs/1.7.2/important-notes/) in the Longhorn documentation.
```

> For information about important changes, including feature incompatibility, deprecation, and removal, see [Important Notes](https://longhorn.io/docs/1.7.1/important-notes/) in the Longhorn documentation.

Reviewer: ```suggestion
For information about important changes, including feature incompatibility, deprecation, and removal, see [Important Notes](https://longhorn.io/docs/1.7.2/important-notes/) in the Longhorn documentation.
```

## Post-Release Known Issues

> For information about issues identified after this release, see [Release-Known-Issues](https://github.com/longhorn/longhorn/wiki/Release-Known-Issues).

Reviewer: Remember to add 1.7.2 section to the WIKI page.
